uiowa/uiowa
# [Bug] padding fix for cards

Resolves https://github.com/uiowa/uiowa/issues/7774.

# How to test

Review https://github.com/uiowa/uids/pull/931.

```
ddev blt frontend && ddev blt ds --site=iowajpec.org && ddev drush @org-iowajpec.local uli /
```

Look at the home page in Safari's "responsive design mode" and verify that stacking looks better than the screenshot posted in the issue above.
